hack/test/e2e.sh

```diff
@@ -49,127 +51,126 @@ PROVISIONER=
 CLUSTER_NAME=
 
 cleanup_capi() {
-  ${KUBECTL} --kubeconfig /tmp/e2e/docker/kubeconfig delete cluster ${NAME_PREFIX}
+  ${KUBECTL} --kubeconfig /tmp/e2e/docker/kubeconfig delete cluster "${NAME_PREFIX}"
 }
 
 # Create a cluster via CAPI.
 function create_cluster_capi {
   trap cleanup_capi EXIT
 
-  ${KUBECTL} --kubeconfig /tmp/e2e/docker/kubeconfig apply -f ${TMP}/cluster.yaml
+  ${KUBECTL} --kubeconfig /tmp/e2e/docker/kubeconfig apply -f "${TMP}/cluster.yaml"
 
   # Wait for first controlplane machine to have a name
-  timeout=$(($(date +%s) + ${TIMEOUT}))
```

Comment:
this dropped `$` looks weird

Reply:
that's intentional, no need of interpolation when using the `$((` syntax

Reply:
https://www.shellcheck.net/wiki/SC2004